RubeesEat/Pages/SplitBill.cshtml

```diff
@@ -0,0 +1,114 @@
+
+    <label for="billPeople">Add Person:</label>
+    <select name="billPeople" id="billPeople">
+        @foreach (Person person in PersonRepository.GetAll())
+        {
+            <option value="@person.FirstName @person.LastName" data-id="@person.Id">@person.FirstName @person.LastName </option>
+        }
+    </select>
+    <input id="addPersonButton" type="button" onclick="AddPerson()" value="Add Person"><br/>
+
+    <button id="splitBillButton" type="button" onclick="validate()">Split the bill</button><br>
+    <text id="placeForErrorMessage" style="color: red;"></text>
+
+    <script type="text/javascript">
+        let counter = 0;
+        function AddPerson() {
+            const selectElement = document.getElementById('billPeople');
+            const selectedPerson = selectElement.value;
+            const selectedId = selectElement.selectedOptions[0].dataset["id"];
+            
+            const personSpan = document.createElement("span");
+            personSpan.textContent = selectedPerson;
+            personSpan.id = "person" + counter;
+            document.querySelector("#addedPeople").appendChild(personSpan);
+            
+            const personIdElement = document.createElement("input");
+            personIdElement.type = "hidden";
+            personIdElement.name = "id" + counter;
+            personIdElement.value = selectedId;
+            document.querySelector("#addedPeople").append(personIdElement);
+            
+            const amountElement = document.createElement("input");
+            amountElement.name = "amount" + counter;
+            amountElement.type = "number";
+            amountElement.step = ".01";
+            document.getElementById("person" + counter).append(amountElement);
+            document.getElementById("person" + counter).append(document.createElement("br"));
+            counter++;
+            
+            const dropdown = document.getElementById("billPeople");
+            dropdown.remove(dropdown.selectedIndex)
+        }
+        
+        function validate() {
+            const description = document.getElementById("billDescription");
+            
+            const amounts = document.getElementById("addedPeople").querySelectorAll("input[name^='amount']");
+            
+            let amountsInvalid = false;
+            let amountsNotPositive = false;
+            amounts.forEach((amount) =>
+            {
+               if (!amount.value)
+                    amountsInvalid = true;
+                if (amount.value <= 0)
+                    amountsNotPositive = true;
+            });
+            if (amountsInvalid) {
+                document.getElementById("placeForErrorMessage").textContent = "Amounts are not valid.";
+                return;
+            }
+            
+            if (amountsNotPositive) {
+                document.getElementById("placeForErrorMessage").textContent = "Amounts should be positive.";
+                return;
+            }
```

Comment:
```suggestion
            amounts.forEach((amount) =>
            {
               if (!amount.value || amount.value <= 0) {
                    document.getElementById("placeForErrorMessage").textContent = "Amounts are not valid.";
                return;
                }
            });
```

Reply:
adjust message to descript what am amount should look like and print the wrong amount

Reply:
We cant do that because return does not end the function

Reply:
We cant always print the wrong amount because the html input element only sends numbers. e or --3 for example cant be depicted. Only negative numbers can be depicted.